napari/_qt/qt_viewer.py

```diff
@@ -285,6 +284,61 @@ def dropEvent(self, event):
                 filenames.append(path)
         self._add_files(filenames)
 
+    def _add_files(self, filenames):
+        """Adds an image/volume layer to the viewer.
+
+        Whether the image is multichannel is determined by
+        :func:`napari.util.misc.is_multichannel`.
+
+        If multiple images are selected, they are stacked along the 0th
+        axis.
+
+        If there is only 1 filename opened and it is a 3D numpy array saved
+        in either npy or npz format, a Volume layer is added and the camera is
+        changed to a 3D camera called TurntableCamera.
+
+        TODO Add event to switch between 3 kinds of cameras that vispy offers
+
+        Parameters
+        -------
+        filenames : list
+            List of filenames to be opened
+        """
+
+        if len(filenames) == 1 and (
+            filenames[0].endswith(".npy") or filenames[0].endswith(".npz")
+        ):
+            volume = load_numpy_array(filenames[0])
+
+            # Create an XYZaxis visual
+            self.axis = XYZAxis(parent=self.view)
+            self.axis.transform = STTransform(
+                translate=(50, 50), scale=(50, 50, 50, 1)
```

Comment:
why the magic numbers of translate (50, 50) and scale (50, 50, 50, 1)? Are these related to the shape of the 3D volume that you're trying to view, in which case they should be computed based on it, or are they independent of it and will work for all volumes? Either way can you add a comment